phpstan.neon.8.1.dist

```diff
@@ -12,3 +12,5 @@ parameters:
         - tests
     excludePaths:
         - vendor
+    ignoreErrors:
+        - '#Method .* has parameter \$context with no value type specified in iterable type array.#'
```

Comment:
Could a note be added to notice that this error could be removed once the dependencies requirements will be forced to 6.2 and higher?

Reply:
Done :+1: